.github/workflows/Weekly_Func_Env_Operator_CI.yml

```diff
@@ -80,6 +80,8 @@ jobs:
           CNV_VERSION: "4.13"
           ODF_VERSION: "4.12"
           NUM_ODF_DISK : "4"
+          KATA_CHANNEL: "stable-1.4"
```

Comment:
Need a comment that this is a temporary workaround here and elsewhere.

Reply:
Its not temporary, once 1.4.1 issue fix I will remove just the value and I will leave all the rest as is

Reply:
Discussed offline and agree with the current wording.